classes/lang/KeysReference/MetaLang.php

```diff
@@ -24,7 +24,7 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 trans('404 error', 'Shop.Navigation');
-trans('Best sales', 'Shop.Navigation');
+trans('Best sellers', 'Shop.Theme.Catalog');
```

Comment:
HI @Progi1984. Why change the catalogue?

Reply:
Because the translation `Best sellers` (title of the page Best Sellers) is existing in an other domain.

Reply:
Ok thanks for your response.